examples/zephyr_native_posix/README.md

```diff
@@ -24,6 +24,14 @@ zephyr/zephyr.exe -testargs -p <i><b>PROJECT_ID</b></i> -d projects/<i><b>PROJEC
 
 ## Troubleshooting
 
+### Installing Dependecies 
+If you're recieving a configuring incomplete error then you might not have the correct compiler installed, to download 
```

Comment:
If you're encountering the error `Configuring incomplete, errors occurred!` then you may be missing build requirements. Install the `device-tree-compiler` and `gperf` packages using the appropriate tools for your environment. For example, in Linux you can use `apt-get` as follows: